packages/manager/apps/nasha-react/src/pages/index.scss

```diff
@@ -0,0 +1,35 @@
+ul {
+  list-style-type: none;
+}
+
+.col {
+  border: 1px solid #e6e6e6;
+  border-radius: 6px;
+  flex: 3;
+  margin: 15px;
+}
+.element {
+  margin: 15px;
+}
+.chipDashboard {
+  float: right;
```

Comment:
We still use float ?

Reply:
It's better to use flexbox/grid to organize the content and use float to integrate images with texts.